extras/ping/ping.go

```diff
@@ -572,15 +585,47 @@ func (p *Pinger) processPacket(recv *packet) error {
 	return nil
 }
 
+func (p *Pinger) parseEchoReplyFromICMP(data []byte, from *net.IP) *Packet {
+	replyPacket, err := icmp.ParseMessage(1, data[:])
+	if err != nil {
+		log.Printf("error: %v\n", err)
```

Comment:
Thank you for using `log` here. I'd like to encourage you to replace `fmt.Printf` with `log.Printf` throughout the code, otherwise we'll have troubles when we integrate this code with `ooniprobe`.